CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # 3.1.0-wip
 
 - new lint: `no_wildcard_variable_uses`
+- fix `prefer_interpolation_to_compose_strings` for false-positive when
```

Comment:
FWIW, I think we're headed away from CHANGELOG entries for fixes of this scope. 

See: https://github.com/dart-lang/linter/pull/4430

Reply:
Ah, OK, I will keep this in mind for the future.